alphastats/loader/MaxQuantLoader.py

```diff
@@ -33,15 +33,15 @@ def __init__(
         self.filter_columns = filter_columns + self.filter_columns
         self.confidence_column = confidence_column
         self.software = "MaxQuant"
-        self.set_filter_columns_to_true_false()
+        self._set_filter_columns_to_true_false()
         if evidence_file is not None:
-            self.load_evidence(evidence_file=evidence_file)
+            self._load_evidence(evidence_file=evidence_file)
 
-    def load_evidence(self, evidence_file, sep = "\t"):
+    def _load_evidence(self, evidence_file, sep="\t"):
         self.evidence_file = pd.read_csv(evidence_file, sep=sep, low_memory=False)
         # check if names match protien group file
 
-    def set_filter_columns_to_true_false(self):
+    def _set_filter_columns_to_true_false(self):
```

Comment:
`to_true_false` is a bit hard to understand? Maybe something like convert_to_bool?